### PR TITLE
Bug 1314708: Test for translation interface

### PR DESCRIPTION
**Pull request by stephaniehobson**

### CODE REVIEW

**Comment by stephaniehobson** [comment]
Oops, forgot to create a PR for this branch. @m8ttyB R?

File: tests/functional/test_article_translate.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import pytest
+
+from pages.article_translate import TranslatePage
+from pages.article import ArticlePage
```
Comment (escattone) [inline tests/functional/test_article_translate.py]: ArticlePage is not used so can remove this line.

File: tests/pages/article_translate.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
+from pypom import Region
```
Comment (escattone) [inline tests/pages/article_translate.py]: `Region` is unused so can remove this line.

**Comment by stephaniehobson** [comment]
Updated 🐨

**Comment by escattone** [comment]
This PR adds one test, and since it's marked with "login", it will not be run as part of the integration tests on staging. With the fixes I've added, it does run against a local development environment using the sample DB, as longs as the sample DB has been modified as follows:
- the passwords for the test user accounts have to be fixed (at least mine did, they were corrupted)
- the page http://localhost:8000/fr/docs/User:anonymous:uitest must be added by translating http://localhost:8000/en-US/docs/User:anonymous:uitest

**Comment by jwhitlock** [comment]
Added some tasks to PR #4076 for the missing and incorrect fixtures.

**Pull request merged**